### DISCUSSION TITLE
Domain Forwarding with Multiple Custom Domains

### DISCUSSION THREAD

**Discussion by valZho** [simple-login/app#2205] [Feature Requests]
For those with multiple custom domains that all just point to the same place, we should be able to set up "domain forwarding" of some sort. Instead of having to recreate every single alias for every domain, a user should be able to just specify that "domain2.com" points to "domain1.com". Alternatively/additionally, when creating an alias, allow the alias to be valid across multiple custom domains.

EXAMPLE:

Imagine you start with:

- alias1@domain1.com
- alias2@domain1.com
- alias3@domain1.com
- ...
- alias100@domain1.com

And now you have a second domain that just points to the first, and you just want to be able to use those domains interchangeably. As I understand it now, you would have to recreate everything above for domain2.com (a lot of work!). Additionally, when you wanted to update/deactivate/delete any alias, you would have to remember to do it in both places.

Now add even more domains — it becomes a nightmare to manage.

**Comment by nguyenkims** [comment]
You can activate catch-all option and the addresses will be automatically created when they receive an email.

**Reply by valZho** [reply]
That doesn't address the issue. First off, what if someone doesn't want any catch-alls (I don't), and secondly, one would still be left managing multiple times the number of aliases that would exist with a domain forward. And even with a catch-alls, any aliases that go to anything but the default email (other people, multiple people) have to be set up ahead of time -- multiply all that work by the number of domains you have. If you are managing a team/family using multiple domains it gets pretty annoying. I know I am an outlier, but it seems like it would be such an easy thing to implement (but maybe not 🤷)
